docs/UserGuide.md

```diff
@@ -194,10 +199,10 @@ Format: `addc c/CLASS_NAME...`
 * Add classes with specified names. 
 * The class names are **case-insensitive**.
   * e.g. If a class with a name **CS1101S** already exists, `addc c/cs1101s` does not add a
-  new class `cs1101s`.
+    new class `cs1101s`.
 * If there are duplicate class names, the class name is taken to be the last valid class parameter `c/`.
   * e.g. If a class with a name **CS1101S** does not exist, `addc c/CS1101S c/cs1101s` adds one class with the name
-  **cs1101s**.
+    **cs1101s**.
```

Comment:
One of cs1101s is in monospace, another in bold. Is there a reason for the difference?

Reply:
Good spot, this should not be the case.